### PR TITLE
LPS-147457 - Improve accessibility errors caused by the use of 'conte…

### PR DESCRIPTION
**Pull request by georgel-pop-lr**
…nt:' from style sheets

### CODE REVIEW

**Comment by georgel-pop-lr** [comment]
ci:test:sf

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	}
 
 	.hide-data-labels .toggle-switch-handle:after {
-		content: normal !important;
+		content: none !important;
```
Comment (diegonvs) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/css/main.scss:33]: Hey @georgel-pop-lr, why this change was necessary? Sorry for the inconvenience here :P

Comment (georgel-pop-lr) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/css/main.scss:33]: Hi @diegonvs, is very similar with the solution @pat270 implemented inside clay https://github.com/liferay/clay/pull/4686/files

Basically, is needed for OAW tool to not detect this as a content added from CSS styles, as this content is considered not accessible by reader tools.

Regards.

Comment (diegonvs) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/css/main.scss:33]: Oh ok!

**Review by diegonvs — APPROVED** [review]

**Comment by diegonvs** [comment]
ci:forward
